dags.yaml

```
@@ -378,3 +378,19 @@ bqetl_desktop_platform:
       ]
     retries: 2
     retry_delay: 30m
+
+bqetl_mozregression:
+  default_args:
+    depends_on_past: false
+    email:
+    - wlachance@mozilla.com
+    - telemetry-alerts@mozilla.com
+    email_on_failure: true
+    email_on_retry: true
+    end_date: null
+    owner: wlachance@mozilla.com
+    retries: 2
+    retry_delay: 30m
+    start_date: '2020-06-01'
+  schedule_interval: daily
```

```suggestion
  schedule_interval: 0 4 * * *
```

`@daily` runs at UTC midnight. This will need to wait on copy_deduplicate which won't be done until ~02:00. Unless there's a need for promptness, I'd recommend waiting until at least 04:00 to make sure this isn't competing for slots with desktop and mobile product ETL.